ui/src/components/inputs/Editor.vue

```diff
@@ -102,7 +102,8 @@
                 oldDecorations: [],
                 editorDocumentation: undefined,
                 plugin: undefined,
-                taskType: undefined
+                taskType: undefined,
+                lastTimeout: undefined,
```

Comment:
lastTimeout must not be a `data` but only a simple props to avoid the reactive part of vue on this value